skeleton-es2016-aspnetcore/src/skeleton-navigation-es2016-vs/Views/Home/Index.cshtml

```diff
@@ -1,12 +0,0 @@
-﻿<div aurelia-app="main">
-    <div class="splash">
-        <div class="message">Aurelia Navigation Skeleton</div>
-        <i class="fa fa-spinner fa-spin"></i>
-    </div>
-
-    <script src="jspm_packages/system.js"></script>
-    <script src="config.js"></script>
-    <script>
-        System.import('aurelia-bootstrapper');
-    </script>
-</div>
```

Comment:
@mikekidder 
I've already made the changes in my local repository (and already fixed the documentation when I realized that it did not mention how to run the solution. There is an existing PR that would add this to the skeleton that also has the changes needed for the new .d.ts strategy. Rob may hold off merging until I have the RTM version though,
I'll add the "dotnet restore" mention. Since I am using Visual Studio this was done automatically for me as was the "npm install". You only have to do the "jspm install -y" if you are using VS.